include/kernels/gauge_force.cuh

```diff
@@ -0,0 +1,178 @@
+#pragma once
+
+#include <gauge_field_order.h>
+#include <quda_matrix.h>
+#include <index_helper.cuh>
+#include <kernel.h>
+#include <shared_memory_cache_helper.cuh>
+
+namespace quda {
+
+  struct paths {
```

Comment:
Use `Path` since it is the convention throughout QUDA?

Reply:
It is?  No one ever told me 😄